Crash on clicking text with strongMorph in KJVA

**Describe the bug**
Crash on clicking text with strongMorph in KJVA.

**Bug was found on And Bible version**
Develop branch (3df322dd2af4b8d6d1f8198a81a5bb9c4d0c9f5f)

**To Reproduce**
1. Open Genesis 1:1 in KJVA
2. Ensure that morphology is on (so phrases are underlined).
3. Click the word created.
4. The application crashes.

**Screenshots**
https://imgur.com/a/5XEmHIY

**Additional content**
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: net.bible.android.activity, PID: 29867
    java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:526)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
     Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950) 
     Caused by: org.crosswire.jsword.passage.NoSuchVerseException: Cannot understand TH as a chapter or verse.
        at org.crosswire.jsword.passage.AccuracyType.parseInt(AccuracyType.java:527)
        at org.crosswire.jsword.passage.AccuracyType.checkValidChapterOrVerse(AccuracyType.java:506)
        at org.crosswire.jsword.passage.AccuracyType.fromText(AccuracyType.java:462)
        at org.crosswire.jsword.passage.AccuracyType.fromText(AccuracyType.java:398)
        at org.crosswire.jsword.passage.VerseRangeFactory.fromText(VerseRangeFactory.java:117)
        at org.crosswire.jsword.passage.VerseRangeFactory.fromString(VerseRangeFactory.java:87)
        at org.crosswire.jsword.passage.VerseRangeFactory.fromString(VerseRangeFactory.java:55)
        at org.crosswire.jsword.passage.AbstractPassage.addVerses(AbstractPassage.java:930)
        at org.crosswire.jsword.passage.RangedPassage.<init>(RangedPassage.java:93)
        at org.crosswire.jsword.passage.PassageType$4.createPassage(PassageType.java:99)
        at org.crosswire.jsword.passage.PassageKeyFactory.getKey(PassageKeyFactory.java:168)
        at org.crosswire.jsword.passage.PassageKeyFactory.getKey(PassageKeyFactory.java:187)
        at net.bible.android.control.link.LinkControl.getBibleKey(LinkControl.kt:170)
        at net.bible.android.control.link.LinkControl.getBookAndKey(LinkControl.kt:99)
        at net.bible.android.control.link.LinkControl.loadApplicationUrl(LinkControl.kt:77)
        at net.bible.android.view.activity.page.BibleView.openLink(BibleView.kt:487)
        at net.bible.android.view.activity.page.BibleView$BibleViewClient.shouldOverrideUrlLoading(BibleView.kt:539)
        at org.chromium.android_webview.AwContentsClientBridge.shouldOverrideUrlLoading(chromium-TrichromeWebViewGoogle.aab-stable-432414133:14)
        at android.os.MessageQueue.nativePollOnce(Native Method)
        at android.os.MessageQueue.next(MessageQueue.java:336)
        at android.os.Looper.loop(Looper.java:174)
        at android.app.ActivityThread.main(ActivityThread.java:7682)
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950) 
```

--- a/app/src/main/java/net/bible/android/view/activity/page/BibleView.kt
+++ b/app/src/main/java/net/bible/android/view/activity/page/BibleView.kt
@@ -374,6 +374,7 @@ class BibleView(val mainBibleActivity: MainBibleActivity,
                 "content" -> "$type:$target"
                 "strong" -> "$type:$target"
                 "robinson" -> "$type:$target"
+                "strongMorph" -> "$type:$target"
                 else -> {
                     if(target.startsWith("sword://"))
                         target

